src/main/java/macrobase/analysis/result/OutlierClassificationResult.java

```diff
@@ -0,0 +1,4 @@
+package macrobase.analysis.result;
+
+public abstract class OutlierClassificationResult {
```

Comment:
The reason `OutlierClassificationResult` is empty is that I haven't defined the API yet, I decided to leave that until we finalize the rest of the API. Because based on what we decide for the other things this will change. If we are happy with the rest of API, let's think about how to do this. I agree that this should not be left empty.

I also though of this class/interface as encompassing all the results that classifier would return, so it would probably have methods more like `getOutliers()`, `getInliers()`, or maybe iterators instead of getters.. I'm open to suggestions.